src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1343,6 +1343,10 @@ private RendezvousAwaitable<int> InternalSendRequestAsync(WinHttpRequestState st
                     state.Dispose();
                     WinHttpException.ThrowExceptionUsingLastError();
                 }
+                int lastError = Marshal.GetLastWin32Error();
```

Comment:
If you're adding code like this....then it needs to be in an ifdef DEBUG.  Because otherwise, it won't be used in Retail builds and you'll get a compile error.